examples/vue/vue-2/src/components/DynamicallyRenderBuilderPage.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div id="home">
+    <div>Hello world from your Vue project. Below is Builder Content:</div>
+
+    <div v-if="canShowContent">
+      <div>
+        page:
+        {{ (content && content.data && content.data.title) || 'Unpublished' }}
+      </div>
+      <builder-render-content
+        model="page"
+        :content="content"
+        :api-key="apiKey"
+        :customComponents="getRegisteredComponents()"
+      />
+    </div>
+  </div>
+</template>
+<script>
+import Vue from 'vue';
+
+import {
+  RenderContent,
+  getContent,
+  getBuilderSearchParams,
+  convertSearchParamsToQueryObject,
+  isEditing,
+  isPreviewing,
+} from '@builder.io/sdk-vue/vue2';
+
+// Register your Builder components
+import HelloWorldComponent from './HelloWorld.vue';
+
+export const REGISTERED_COMPONENTS = [
+  {
+    component: HelloWorldComponent,
+    name: 'Hello World',
+    canHaveChildren: true,
+    inputs: [
+      {
+        name: 'text',
+        type: 'string',
+        defaultValue: 'World',
+      },
+    ],
+  },
+];
+
+// TODO: enter your public API key
+const BUILDER_PUBLIC_API_KEY = 'f1a790f8c3204b3b8c5c1795aeac4660'; // ggignore
+
+export default Vue.extend({
+  name: 'DynamicallyRenderBuilderPage',
+  components: {
+    'builder-render-content': RenderContent,
+  },
+  data: () => ({
+    canShowContent: false,
+    content: null,
+    apiKey: BUILDER_PUBLIC_API_KEY,
+  }),
+  methods: {
+    getRegisteredComponents() {
+      return REGISTERED_COMPONENTS;
+    },
+  },
+  mounted() {
+    getContent({
+      model: 'page',
+      apiKey: BUILDER_PUBLIC_API_KEY,
+      options: getBuilderSearchParams(
+        convertSearchParamsToQueryObject(new URLSearchParams(window.location.search))
+      ),
+      userAttributes: {
+        urlPath: window.location.pathname,
+      },
+    }).then(res => {
+      this.content = res;
+      this.canShowContent = this.content || isEditing() || isPreviewing();
```

Comment:
thought we had a convo about we were going to purely use `isPreviewing()`? that's the intent of the new SDKs, really need things to be as simple as possible

Reply:
(aka no more need for isEditing() - the editor is also a preview)

Reply:
They aren't quite the same thing: 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/1393142/177594768-b9955e50-671b-4084-a937-396df2e64097.png">

But I agree with you on simplifying this as well. Just requires some more changes. We can also bake this logic into `RenderContent` and remove the need to export these 2 helpers altogether